### ISSUE TITLE
Unable to change Dialog button color

### ISSUE THREAD

**Issue by mg-ahir** [pedant/sweet-alert-dialog#90]
i can't find any method that change dialog button color , because success dialog button show like disable button show , i unable to change button color

**Comment by mzgs2** [comment]
Just add this line your strings.xml and change colors

```
<color name="blue_btn_bg_color">#27ae60</color>
<color name="blue_btn_bg_pressed_color">#2ecc71</color>

<color name="gray_btn_bg_color">#e74c3c</color>
```

**Comment by SuperGirlAndroid** [comment]
Hi,I need to add method for change color of button ,can you help me please any idea ???

**Comment by mzgs2** [comment]
I think it is not possible.
http://stackoverflow.com/a/9585075

**Comment by iman2420** [comment]
How did you implement library in your project?
If you use this code, you can not make any changes:

```

repositories {
    mavenCentral()
}

dependencies {
    compile 'cn.pedant.sweetalert:library:1.3'
}

```

Unless the [Library folder](https://github.com/pedant/sweet-alert-dialog/tree/master/library/src/main) and all of its components in your project and then edit the copy.

**Comment by SuperGirlAndroid** [comment]
No,Ithink it's possible for me I import the code soure in my project and I was modified many things ,but now I need to add a method where I can  to change the color of button without change the others style of button .
I want change just the color not the style .
please do you have any idea ??
